scripts/build.py

```diff
@@ -566,30 +565,34 @@ def _execute_tasks(tasks, batch_size=24):
             task = remaining_tasks.pop()
             currently_running_tasks.append(task)
             task.start()
-        time.sleep(1)
 
 
-def build_files(source, target, file_hashes):
-    """Minifies all CSS and JS files, removes whitespace from HTML and
-    interpolates paths in HTML to include hashes in source
-    directory and copies it to target.
+def build_files(source, target, file_hashes, file_formats=None):
+    """If no specific file formats is provided, minifies all CSS and JS files,
+    removes whitespace from HTML and interpolates paths in HTML to include
+    hashes in source directory and copies it to target.
 
     Args:
         source: str. Path relative to /oppia directory of directory
             containing files and directories to be copied.
         target: str. Path relative to /oppia directory of directory where
             to copy the files and directories.
         file_hashes: dict(str, str). Dictionary of file hashes.
+        file_formats: tuple(str). Tuple of specific file formats to be built.
```

Comment:
file_formats: tuple(str) or None. Tuple of specific file formats to be built. If None, then ...

Reply:
Added more details about when no file format is provided.